mllib/src/main/scala/org/apache/spark/mllib/rdd/MLPairRDDFunctions.scala

```diff
@@ -40,7 +40,11 @@ class MLPairRDDFunctions[K: ClassTag, V: ClassTag](self: RDD[(K, V)]) extends Se
    * @return an RDD that contains the top k values for each key
    */
   def topByKey(num: Int)(implicit ord: Ordering[V]): RDD[(K, Array[V])] = {
-    self.aggregateByKey(new BoundedPriorityQueue[V](num)(ord))(
+      topByKey(num, 16)
```

Comment:
To be clear, was this 16 by default before this PR? Adding a parameter would be fine but this should not change the original behaviour without this parameter before this PR.